lib/loader/index.js

```diff
@@ -265,6 +265,7 @@ function demangle(exports, baseModule) {
       let classElem = curr[className];
       if (typeof classElem === "undefined" || !classElem.prototype) {
         let ctor = function(...args) {
+          args.unshift(null); // pass null pointer for class constructor of AssemblyScript
           return ctor.wrap(ctor.prototype.constructor(...args));
```

Comment:
Would be a bit more efficient to write as `return ctor.wrap(ctor.prototype.constructor(0, ...args));` I think